src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -404,6 +404,9 @@ def _open(self):
         elif 'frontera' in self.rm.host.lower():
             self._ppn = 56
 
+        elif 'comet' in self.rm.host.lower():
```

Comment:
Why is there a change on slurm? This is not cobalt relevant?

Reply:
This is because of an out-of-sync merge, will be resolved in devel.  Thanks for noticing.